### ISSUE TITLE
Data and resource file re-org

### ISSUE THREAD

**Issue by chengzhuzhang** [E3SM-Project/e3sm_to_cmip#85]
e3sm_to_cmip uses data from another git repo as input:
Meta data definition files from https://github.com/E3SM-Project/CMIP6-Metadata
and
cmor tables https://github.com/PCMDI/cmip6-cmor-tables

A version of cmor tables are copied over to 
e3sm_to_cmip/e3sm_to_cmip/resources/

A re-org of these data files are needed.

**Comment by chengzhuzhang** [comment]
@xylar we will need data from above two repos if we were include cmorizing in zppy. I'm thinking maybe we can address this through either e3sm_unified or mache?

**Comment by xylar** [comment]
@chengzhuzhang, I don't think that E3SM-Unified or mache provides a clear path for including these other repos.

I think you would want to make recipes to make conda packages of these repos (this would require licenses for both and a release for CMIP6-Metadata).  Then, they could be dependencies of `e3sm_to_cmip`.

A conceptually similar conda package is `cartopy_offlinedata`:
https://github.com/willirath/cartopy_offlinedata/blob/master/bin/download_cartopy_data.py
https://github.com/conda-forge/cartopy_offlinedata-feedstock/blob/master/recipe/build.sh#L7

It downloads data commonly needed by the `cartopy` package and puts it in `share/cartopy/` within the conda environment.  We could do something similar for the data from these two repos.

**Comment by xylar** [comment]
@chengzhuzhang, did you also see this comment at:
https://github.com/PCMDI/cmip6-cmor-tables
> Only CMOR and PrePARE should use the JSON file of concatenated CMIP6 Control Vocabulary CMIP6_CV.json found in this repository.
>
> All other software should get the CV's directly from the WCRP repository.

So should `e3sm_to_cmip` be using https://github.com/WCRP-CMIP/CMIP6_CVs instead?

**Comment by chengzhuzhang** [comment]
@xylar Hey, thank you for the suggestions. Unlike the catopy-offline data. The two repos concerned are less popular. Not sure if it worth the effort to do the conversion, with potential license problems on the side. It might be worthwhile to make the cmip6-cmor-tables a conda package. But, https://github.com/E3SM-Project/CMIP6-Metadata will be mostly used internally within E3SM. On possibility I'm thinking is that, we touched on using `mache` for consolidating managing paths for diagnostics data cross machines, data in these two repos can be handled similarly?

**Comment by chengzhuzhang** [comment]
`e3sm_to_cmip` uses CMOR, so `cmip6-cmor-tables` should be the right repo.

**Comment by xylar** [comment]
@chengzhuzhang, thanks for the further discussion.  Thanks for clarifying about `cmip6-cmor-tables`.

I also get why having these data repos as conda packages on conda-forge probably isn't the way to go.

However, I'm wondering if `mache` is really the way to go.  The only reason to try to include this data in `mache` would be if we plan to download it manually onto E3SM supported machines, and `mache` would just contain information about where that data had been downloaded.  But wouldn't it be far preferable to have the `e3sm_to_cmip` package download the data into the conda environment?  Then, different versions of `e3sm_to_cmip` can point to different versions of the data git repos.  Perhaps more importantly, people could use `e3sm_to_cmip` on non-supported machines.
